packages/coze-uniapp/src/types.ts

```
@@ -0,0 +1,51 @@
+/**
+ * 事件系统类，用于替代UniApp的Events
+ */
+export class Events {
+  private events: Record<string, Function[]> = {};
+
+  on(eventName: string, callback: Function) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)